manifest.json

```diff
@@ -507,11 +507,11 @@
         }
       },
       "uiCodemirror": {
-        "version": "0.3.0",
+        "version": "",
         "downloadFormat": "zip",
-        "url": "https://github.com/angular-ui/ui-codemirror/archive/src0.3.0.zip",
-        "rootDirPrefix": "ui-codemirror-src",
-        "targetDirPrefix": "ui-codemirror-"
+        "url": "https://github.com/oppia/ui-codemirror/archive/master.zip",
```

Comment:
Nope -- use a specific version. Never use master/develop/etc. -- it can update under you and break things. Use a specific version so that everyone can depend on the same stable URL.